### PR TITLE
Speed keras combo

### PR DESCRIPTION
**Pull request by emlynjdavies**
Experimental merge of speed-test branch and updated pandas branch.

### CODE REVIEW

File: pysilcam/silcam_classify.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import h5py
 import numpy as np
 import pandas as pd
-import scipy
+from PIL import Image
+import tensorflow as tf
```
Comment (tankefugl) [inline pysilcam/silcam_classify.py:7]: F401: 'tensorflow as tf' imported but unused

**Review by tankefugl — DISMISSED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Comment by emlynjdavies** [comment]
- [ ] GUI DLL problem on windows

**Comment by emlynjdavies** [comment]
On windows:

* Initializing background image handler
* Commencing image acquisition and processing
2.8295562267303467 Acquisition loop time
1.3208818435668945 Acquisition loop time
1.4181203842163086 Acquisition loop time
Process Process-1:
Traceback (most recent call last):
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 64, in <module>
    from tensorflow.python._pywrap_tensorflow_internal import *
ImportError: DLL load failed while importing _pywrap_tensorflow_internal: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\keras\__init__.py", line 3, in <module>
    from tensorflow.keras.layers.experimental.preprocessing import RandomRotation
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\__init__.py", line 41, in <module>
    from tensorflow.python.tools import module_util as _module_util
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\python\__init__.py", line 40, in <module>
    from tensorflow.python.eager import context
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\python\eager\context.py", line 35, in <module>
    from tensorflow.python import pywrap_tfe
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\python\pywrap_tfe.py", line 28, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 83, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Users\emlynd\Anaconda3\envs\silcam\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 64, in <module>
    from tensorflow.python._pywrap_tensorflow_internal import *
ImportError: DLL load failed while importing _pywrap_tensorflow_internal: The specified module could not be found.


Failed to load the native TensorFlow runtime.

**Review by nepstad — APPROVED** [review]

**Review by WillRNaylor — APPROVED** [review]

**Pull request merged**